packages/link/Link.js

```diff
@@ -62,6 +62,7 @@ const AvLink = ({
     <Tag
       href={url}
       target={target}
+      style={{ textDecoration: 'underline' }}
```

Comment:
We should move this style object to the outer scope so that it isn't recreated on every render/update

This is a good multi-part article explaining the cause of renders and common fixes medium.com/welldone-software/why-did-you-render-mr-big-pure-react-component-2a36dd86996f